products/dns-security/docs/examples/request-domain-categorization-change.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: request-domain-categorization-change
+title: Request Domain Categorization Change
+description: DNS Security API example showing how to submit a change request for a domain's categorization.
+slug: /threat-vault/docs/examples/dns-security/request-domain-categorization-change
+hide_title: false
+hide_table_of_contents: false
+keywords:
+  - security subscription
+---
+
+The API Reference information for retrieving domain information can be found [here](/cdss/dns-security/api/dns-security#operation/PublicApi_V1DomainChangeRequest).
```

Comment:
This link is no longer valid with the new API Docs. It should be at the path `/dns-security/api/public-api-v-1-domain-info`